core/snippets/snippets/markdown.snippet

```diff
@@ -4,5 +4,48 @@ language: markdown
 name: link
 phrase: link
 -
-[$1]($0)
+[$0]($1)
+---
+
+name: image
+phrase: image
+-
+![$0]($1)
```

Comment:
Because $0 is for the last snippet hole, this goes from right to left. Is there a reason to do that with this snippet?